src/IntrType.f90

```diff
@@ -60,6 +61,8 @@ MODULE IntrType
 PUBLIC :: OPERATOR(.APPROXEQR.)
 PUBLIC :: OPERATOR(.APPROXLE.)
 PUBLIC :: OPERATOR(.APPROXGE.)
+PUBLIC :: OPERATOR(.APPROXEQDS.)
```

Comment:
The existing ones have been extended to take a double and a single, but I couldn't see a way to do two doubles with single precision tolerance under the existing interfaces. It is possible to downscale doubles to singles and use the existing interface, but there are hundreds of failing checks that need downscaling. So adding an operator is a more convenient way as I understand the idea of defining these operators here. I would expect this to be often used later if we switch to mixed precision.

Reply:
I guess my expectation is that we really should be using soft equivalance instead of approxeq if we're really saying that single precision is okay.  I just don't understand why this is really needed... but in many cases, I don't really like approxeq in general.

Reply:
Year there could be two ways here:
1. Using the new approxeqds.
2. Use softeq but will have to make them all go with the FSCALE factors in order for both double and mixed precisions to work.
If the second approach is preferred, I can make the change. This will eliminate the use of the flag MIXPRECISION in Futility.